platform/darwin/test/MGLStyleLayerTests.mm.ejs

```diff
@@ -59,6 +59,7 @@
     MGLTransition transitionTest = MGLTransitionMake(5, 4);
 
 <% for (const property of properties) { -%>
+<%   if (property.name === 'heatmap-color') continue; -%>
```

Comment:
Note that I just bailed out of generating tests here.  I'm thinking it makes sense to just manually create an analogous unit test for `heatmapColor` -- @1ec5 does that sound right to you?  If so, where/how's the best way to do that?

Reply:
Yes, that’s a fine idea: create a separate file alongside MGLHeatmapStyleLayerTests.mm – like MGLHeatmapColorTests.mm – and add it to the test target of ios.xcodeproj and macos.xcodeproj. Then import the XCTest umbrella header and implement a subclass of XCTestCase.